_episodes/06-style-guide.md

```diff
@@ -3,13 +3,23 @@ title: "Style Guide"
 teaching: 10
 exercises: 0
 questions:
+- "What style should be used for Carpentries lessons?"
 - "How are keyboard key combinations written?"
+- "How are short spans of code written?"
 objectives:
 - "Explain keyboard key combination."
+- "Locate documentation for general matters of style in The Carpentries."
+- "Identify uses of the span of code Markdown feature."
 keypoints:
-- "Keyboard keys need to use `<kbd>` HTML tag."
+- "The Carpentries Handbook contains a style guide."
+- "Keyboard actions are described using the `<kbd>` HTML tag."
+- "A backtick (_\`_) indicates a span of code and is used to enclose code excerpts, segments of text to type, and 
```

Comment:
@maxim-belkin is this more clear?

Reply:
Ha! I was actually commenting here at this very moment.

--- 
This objective causes rendering issues. Please add one more backslash to escape the backtick.

On a different point: we use backticks not to enclose something (code excerpts, etc.) but we enclose something within backticks to indicate that what we enclose within backticks is a code excerpt, etc....

I suggest rephrasing to something like this:

```
- "Enclosing text in backtics \\` indicates that this text is a code excerpt, 
   segment of text to type, or a filename."
```